1-exercises/A-accessing-values/exercise3.js

```diff
@@ -20,7 +20,10 @@ let basketballTeam = {
     - console.logs the name of each player on a new line
 */
 
-// write code here
+let players = basketballTeam.topPlayers.sort();
```

Comment:
instead of calling the method just players you could write 'sortedPlayers' to add more of a description to what the method is doing.